charts/cray-spire/templates/server/configuration.yaml

```diff
@@ -217,6 +217,7 @@ data:
     create_if_new compute ckdump_helper /opt/cray/cray-spire/ckdump_helper 864000
     create_if_new compute cpsmount /opt/cray/cray-spire/cpsmount-spire-agent
     create_if_new compute cpsmount_helper /opt/cray/cps-utils/bin/cpsmount_helper
+    create_if_new compute cos-config-helper /opt/cray/cray-spire/cos-config-helper
```

Comment:
The paths in configuration don't match the paths in workloads. Which one is the path you're using?

Reply:
Fixed.  Thanks.